src/app/pages/Collections/Collections.tsx

```diff
@@ -128,9 +143,33 @@ export default function Collections(): JSX.Element {
               searchValue={addNewCollection}
               onChange={(event) => setAddNewCollection(event.target.value)}
             />
+            <AddButton
+              className={styles.addButtonPos}
+              type="button"
+              onClick={() => setModalToggle(false)}
+            >
+              Close
+            </AddButton>
           </form>
         </section>
       )}
+      {modalToggleDelete && (
+        <section className={styles.modal} id="modal">
+          <div className={styles.warning}>
+            <p>DELETE</p>
+            <p className={styles.titleColor}>" {deleteCollection.name} "</p>
+            <p>Are you sure</p>
+            <div className={styles.buttonWrapperModal}>
+              <AddButton onClick={() => handleDeleteClick(deleteCollection)}>
+                Yes
+              </AddButton>
+              <AddButton onClick={() => setModalToggleDelete(false)}>
+                No
+              </AddButton>
```

Comment:
The `AddButton` has a less fitting name, since it seems too versatile. Isn't it just a `Button` or `ActionButton`?